api/src/main/java/org/eclipse/microprofile/metrics/Metadata.java

```diff
@@ -63,421 +61,83 @@
  *
  * @author hrupp, Raymond Lam
  */
-public class Metadata {
+public interface Metadata {
```

Comment:
copyright year should be 2017, 2018 on this file.  

From https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated ... 

> Therefore, if the claim is based on common law copyright (not formally registered), then the date should be the date of first publication. If the claim is a registered copyright, then the date should be the date claimed in the registration. In cases where the work was substantially revised you may establish a new copyright claim to the revised work by adding another copyright notice with a newer date or by adding an additional date to the existing notice as in "© 2000, 2010". Again, the added date establishes how far back the claim is made on the revision.